packages/components/tests/dummy/app/templates/components/app-footer.hbs

```diff
@@ -0,0 +1,206 @@
+      <Hds::AppFooter::StatusLink @status="OPERATIONAL" />
+      <Hds::AppFooter::Link @href="https://cloud.hashicorp.com/docs/changelog">Changelog</Hds::AppFooter::Link>
+      <Hds::AppFooter::LegalLinks />
+      <Hds::AppFooter::Copyright />
+    </Hds::AppFooter>
+
+    <Shw::Divider />
+
+    <Shw::Text::H2>Base components & child components</Shw::Text::H2>
+
+    <button type="button" {{on "click" this.toggleHighlight}}>
+      {{if this.showHighlight "Hide" "Show"}}
+      layout highlight
+    </button>
+
+    <Shw::Text::H3>Hds::AppFooter::Item</Shw::Text::H3>
+
+    <ul class="hds-app-footer__list">
+      <Hds::AppFooter::Item><Hds::Text::Body
+          @tag="span"
+          @size="100"
+          @color="regular"
+        >Item</Hds::Text::Body></Hds::AppFooter::Item>
+    </ul>
+
+    <Shw::Divider @level={{2}} />
+
+    <Shw::Text::H3>Hds::AppFooter::Link</Shw::Text::H3>
+
+    <Shw::Text::H4>States</Shw::Text::H4>
+
+    <Shw::Flex {{style gap="2rem" justifyContent="space-between"}} as |SF|>
+      {{#each @model.STATES as |state|}}
+        <SF.Item @label={{state}}>
+          <ul class="hds-app-footer__list">
+            <Hds::AppFooter::Link @href="#" mock-state-value={{state}}>Link</Hds::AppFooter::Link>
+          </ul>
+        </SF.Item>
+      {{/each}}
+    </Shw::Flex>
+
+    <Shw::Divider @level={{2}} />
+
+    <Shw::Text::H3>Hds::AppFooter::StatusLink</Shw::Text::H3>
+
+    <Shw::Text::H4>Status variants</Shw::Text::H4>
+
+    <Shw::Flex {{style gap="2rem" justifyContent="space-between"}} as |SF|>
+      {{#each-in @model.STATUSES as |status|}}
+        <SF.Item @label={{status}}>
+          <ul class="hds-app-footer__list">
+            <Hds::AppFooter::StatusLink @status={{status}} />
+          </ul>
+        </SF.Item>
+      {{/each-in}}
+    </Shw::Flex>
+
+    <Shw::Text::H4>With custom values</Shw::Text::H4>
+
+    <Shw::Text::Body><strong>Custom values combined with a standard status</strong></Shw::Text::Body>
+
+    <Shw::Flex {{style gap="2rem"}} as |SF|>
+      <SF.Item @label="with custom text">
+        <ul class="hds-app-footer__list">
+          <Hds::AppFooter::StatusLink @status="OPERATIONAL" @text="Operativo" />
+        </ul>
+      </SF.Item>
+
+      <SF.Item @label="with custom icon color">
+        <ul class="hds-app-footer__list">
+          <Hds::AppFooter::StatusLink @status="OPERATIONAL" @statusIconColor="purple" />
+        </ul>
+      </SF.Item>
+
+      <SF.Item @label="with custom icon">
+        <ul class="hds-app-footer__list">
+          <Hds::AppFooter::StatusLink @status="OPERATIONAL" @statusIcon="cloud-check" />
+        </ul>
+      </SF.Item>
+    </Shw::Flex>
+
+    <Shw::Text::Body><strong>Custom values with no status</strong></Shw::Text::Body>
+    <Shw::Flex {{style gap="2rem"}} as |SF|>
+      <SF.Item @label="with custom icon and text">
+        <ul class="hds-app-footer__list">
+          <Hds::AppFooter::StatusLink @statusIcon="hashicorp" @text="HashiCorp" />
```

Comment:
I would keep this example more generic, otherwise one would interpret this as the "copyright" element.

```suggestion
          <Hds::AppFooter::StatusLink @statusIcon="hexagon-fill" @text="Lorem ipsum" />
```